dist/components/tabs/stencil-tabs.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function() {
+    var init = function() {
+        // Set up initial state for accessibility
+        var $tabButtons = $('.c-tabs__button');
```

Comment:
No problem. It is different from what we’re used to, and it’s not that we’re dropping `js-` classes in all cases, so it bears repeating.